arch/X86/X86MappingInsn.inc

```diff
@@ -732,7 +732,7 @@
 },
 
 {
-	X86_ADD_FPrST0, X86_INS_FADD, 0,
+	X86_ADD_FPrST0, X86_INS_FADDP, 0,
```

Comment:
why this change? it seems unrelated to UD0 and UD1

Reply:
I don't really know why that is different. That file is generated. In fact, there is faddp in one of the X86 td files: capstone/suite/synctools/tablegen/X86/X86InstrFPStack.td:def ADD_FPrST0  : FPrST0PInst<MRM0r, "faddp\t$op">;

Reply:
BTW, FYI, I plan to make more updates on this tomorrow.

Reply:
I couldn't find anything about faddp in the git log history: i.e., there's no commit mentioning faddp.